<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyMoneyOrgaGui</class>
 <widget class="QMainWindow" name="PyMoneyOrgaGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyMoneyOrga</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabOrganizeAccounts">
        <attribute name="title">
         <string>Organize Expenses</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>1071</width>
           <height>571</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="1">
           <widget class="QLineEdit" name="inputAddExpenses"/>
          </item>
          <item row="6" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="buttonAddExpenses">
            <property name="text">
             <string>new expenses</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="inputAddIncome"/>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="buttonAddIncome">
            <property name="text">
             <string>new income</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="7">
           <widget class="QTableView" name="tableViewAccounts">
            <property name="showGrid">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboChooseAccount">
            <item>
             <property name="text">
              <string>NoAccountSaved</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelChooseAccount">
            <property name="text">
             <string>choose account</string>
            </property>
            <property name="buddy">
             <cstring>comboChooseAccount</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelAddExpenses">
            <property name="text">
             <string>amount</string>
            </property>
            <property name="buddy">
             <cstring>inputAddExpenses</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="7">
           <widget class="QTableWidget" name="tableWidgetTransactions"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_New_Account"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_New_Account">
   <property name="text">
    <string>Create New Account</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
